js/.eslintrc.cjs

```diff
@@ -0,0 +1,159 @@
+module.exports = {
```

Comment:
Instead of copying the config in every repo, we can use [shareable config package](https://eslint.org/docs/latest/extend/shareable-configs)  (same for prettier)
Wdyt?

Reply:
Could be nice to share the same config between moon-landing and hub-docs !